docs/ldap/ldapserver.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Ldapserver
+description: How to connect ldap client in Casdoor
+keywords: [ldapserver]
+---
+
+Many systems like `Nexus` support `ldap` authentication. A simple ldap server is also implemented in Casdoor, which supports bind and search operations. 
+
+The following describes how to connect to the ldap server in Casdoor and implement simple login authentication.
+
+### LDAP Server Port
+
+The LDAP server listens on port `389` by default, you can change the default port by changing `ldapServerPort` in [conf/app.conf](https://github.com/casdoor/casdoor/blob/28b381e01eebac66e39e20179ed95282695ecd75/conf/app.conf#L22).
+
+### How it works
+
+Similar to the ldap client in Casdoor, the users in the ldap server are all subclasses of `poxisAccount`.
+
+When the server receives a set of data transmitted by the ldap, it will parse the `cn` and `ou`, where `cn` represents the username, `ou` represents the organization name. It doesn't matter what `dc` is.
+
+If it is a bind operation, the server will use Casdoor to verify the username and password and give the user permission in Casdoor.
+
+If it is a search operation, the server will check whether the Search operation is legal according to the permissions given to the client by the bind operation and return a response.
+
+> **We only support `Simple Authentication`**.
```

Comment:
Maybe use [Admonition](https://docusaurus.io/docs/markdown-features/admonitions) is better
like:
```md
:::caution

We only support `Simple Authentication`.

:::
```
or
```md
:::info

We only support `Simple Authentication`.

:::
```

![image](https://user-images.githubusercontent.com/53544726/192155211-a6ffdd62-ab82-461f-a6b7-d65971fb78f0.png)